src/Uno.UWP/Globalization/NumberFormatting/Extensions.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Windows.Globalization.NumberFormatting
```

Comment:
If possible, can you switch the newly created code files to use file-scoped namespaces 👍 ?

Reply:
Sure